velox/exec/tests/StreamingAggregationTest.cpp

```diff
@@ -385,22 +422,17 @@ TEST_F(StreamingAggregationTest, sortedAggregations) {
 }
 
 TEST_F(StreamingAggregationTest, distinctAggregations) {
-  auto data = makeRowVector({
-      makeFlatVector<int64_t>(1'000, [](auto row) { return row / 4; }),
-      makeFlatVector<int64_t>(1'000, [](auto row) { return row % 3; }),
-  });
+  auto size = 128;
 
-  auto plan = PlanBuilder()
-                  .values({data})
-                  .streamingAggregation(
-                      {"c0"},
-                      {"array_agg(distinct c1)"},
-                      {},
-                      core::AggregationNode::Step::kSingle,
-                      false)
-                  .planNode();
+  std::vector<VectorPtr> keys = {
+      makeFlatVector<int32_t>(size, [](auto row) { return row; }),
```

Comment:
Are we testing with single-row groups only? Is this intentional?

Reply:
No, I just copied it from other tests. I will add multikey test cases.